### DISCUSSION TITLE
Serializing to bytes

### DISCUSSION THREAD

**Discussion by davetapley** [yukinarit/pyserde#475] [Ideas]
In the spirit of https://github.com/yukinarit/pyserde/issues/450 I'd like to propose a feature to optionally serialize JSON to `bytes`.

However: `pyserde` supports both `json` and `orjson` ([here](https://github.com/yukinarit/pyserde/blob/24712178fdeb80a8b8b67415ea092aea57e33b51/serde/json.py#L13-L36)), and [`json`](https://docs.python.org/3/library/json.html#basic-usage) says:

> The json module always produces str objects, not bytes objects.

So: How do you feel about a serializer which would do `.encode('utf-8')` in the absence of `orjson`,
and use `orjson.dumps` otherwise?

**Comment by yukinarit** [comment]
Hi @davetapley 
I am fine with adding such an optional parameter to `to_json`!

**Reply by davetapley** [reply]
Great! I'll get a PR in ASAP.
